### PR TITLE
CuboCore.packages: added

### PR DESCRIPTION
**Pull request by dan4ik605743**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dan4ik605743** [comment]
@SuperSandro2000 Well, something like this.

**Comment by dan4ik605743** [comment]
Patch on libcprime took https://github.com/nixos/nixpkgs/pull/131748

**Comment by dan4ik605743** [comment]
@SuperSandro2000 please check my PR

**Comment by dan4ik605743** [comment]
@SuperSandro2000 please check my PR

**Comment by iAmMrinal0** [comment]
Result of `nixpkgs-review pr 131825` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

**Comment by dan4ik605743** [comment]
@SuperSandro2000 please check my PR

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please do not use libsForQt5 as inputs. It should always libsForQt5.callPackage be used and the libraries in the inputs.

File: pkgs/top-level/cubocore-packages.nix [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{ makeScope, newScope, lxqt }:
```
Comment (SuperSandro2000) [inline pkgs/top-level/cubocore-packages.nix:1]: This file is not called anywhere.

**Comment by dan4ik605743** [comment]
> Please do not use libsForQt5 as inputs. It should always libsForQt5.callPackage be used and the libraries in the inputs.

I need my mkDerivation to be used

File: pkgs/top-level/cubocore-packages.nix (hunk 1 above)
Comment (dan4ik605743) [reply pkgs/top-level/cubocore-packages.nix]: I'm just the first time to have a deal with scoop. Sorry if that for mistakes.

**Review by dan4ik605743 — COMMENTED** [review]
I want in the end it turned out.
CuboCore.corefm (Package)

**Review by dan4ik605743 — COMMENTED** [review]
image
Everything is successful.

**Comment by dan4ik605743** [comment]
And oh, I forgot to remove LibcPrime, libcsys. From all-packetages.nix. Now everything will fix it.

**Comment by dan4ik605743** [comment]
I accidentally pulled someone else's commit. I rearrange PR.

**Pull request closed without merging**